Detectors/TRD/simulation/src/Digitizer.cxx

```diff
@@ -316,7 +311,11 @@ bool Digitizer::convertHits(const int det, const std::vector<Hit>& hits, SignalC
           zz = 0.5 - zz;
         }
         // Use drift time map (GARFIELD)
-        driftTime = mDriftEstimators[thread].TimeStruct(driftVelocity, 0.5 * AmWidth - 1.0 * locTd, zz) + hit.GetTime();
+        bool errFlag = false;
+        driftTime = mDriftEstimators[thread].timeStruct(driftVelocity, 0.5 * AmWidth - 1.0 * locTd, zz, &errFlag) + hit.GetTime();
+        if (errFlag) {
+          mFlagVdriftOutOfRange[det] = true;
+        }
```

Comment:
What about dropping the `errFlag` variable and the if-check just by using `&mFlagVdriftOutOfRange[det]` instead of `errFlag`?

Reply:
Good point